kyuubi-common/src/main/scala/org/apache/kyuubi/package.scala

```diff
@@ -73,4 +73,8 @@ package object kyuubi {
   val BUILD_USER: String = BuildInfo.user
   val REPO_URL: String = BuildInfo.repoUrl
   val BUILD_DATE: String = BuildInfo.buildDate
+
+  private def getProperty(props: Properties, key: String, defaultValue: String): String = {
+    Option(props.getProperty(key, defaultValue)).filterNot(_.isEmpty).getOrElse(defaultValue)
+  }
```

Comment:
we don't need to make it generic, move it to `BuildInfo` body and simply write as

```suggestion
  private def getProperty(key: String, defaultValue: String = unknown): String = {
    Option(props.getProperty(key, defaultValue)).filterNot(_.isEmpty).getOrElse(defaultValue)
  }
```

Reply:
I agree, thank you.